### PR TITLE
:sparkles: Adding blabla to PyPI

### PR DESCRIPTION
**Pull request by abhisheknovoic**
This is a PR to make changes to the setup.py file and setup.cfg file to upload BlaBla to PyPI.

Files Changed:

a. setup.py - Added reference to the tar.gz file containing the v0.1 release of BlaBla.
b. setup.cfg - Added reference to the README.md file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,11 +39,18 @@
         "ipython==7.13.0",
         "jsonschema==3.2.0",
         "pyyaml==5.3.1",
-        "pandas==1.0.3",
-        "tqdm==4.46.0"
```
Comment (jackanth) [inline setup.py]: Why have we ditched the tqdm dependency?

File: setup.py [hunk 2]
```diff
@@ -25,11 +39,18 @@
...
+        "pandas==1.0.3"
     ],
     package_data={"": ["*.txt"]},
     include_package_data=True,
     scripts=["bin/blabla"],
     zip_safe=False,
+    classifiers=[
+    'Development Status :: 4 - Beta',      # Chose either "3 - Alpha", "4 - Beta" or "5 - Production/Stable" as the current state of your package
+    'Intended Audience :: Developers',      # Define that your audience are developers
+    'Topic :: Software Development :: Build Tools',
```
Comment (jackanth) [inline setup.py]: Not sure this is the most appropriate topic ('build tools' says to me GNU make/Ninja etc). Something ML/clinical/scientific would be better.

File: setup.py [hunk 3]
```diff
@@ -25,11 +39,18 @@
...
-        "tqdm==4.46.0"
+        "pandas==1.0.3"
     ],
     package_data={"": ["*.txt"]},
     include_package_data=True,
     scripts=["bin/blabla"],
     zip_safe=False,
+    classifiers=[
+    'Development Status :: 4 - Beta',      # Chose either "3 - Alpha", "4 - Beta" or "5 - Production/Stable" as the current state of your package
+    'Intended Audience :: Developers',      # Define that your audience are developers
```
Comment (jackanth) [inline setup.py]: Are developers the intended audience? Recommend `Intended Audience :: Healthcare Industry' and `Intended Audience :: Science/Research' instead.

**Comment by jackanth** [comment]
Let's also replace the current pip install method with `pip install blabla` in the README 😄

File: setup.py (hunk 2 above)
Comment (abhisheknovoic) [reply setup.py]: Have replaced this with "Topic :: Scientific/Engineering :: Bio-Informatics" now.

File: setup.py (hunk 3 above)
Comment (abhisheknovoic) [reply setup.py]: Have replaced this with "Intended Audience :: Healthcare Industry" now

**Comment by abhisheknovoic** [comment]
> Let's also replace the current pip install method with `pip install blabla` in the README 😄

@jackanth , I have edited the README and added the above.

File: setup.py [hunk 4]
```diff
@@ -32,4 +46,12 @@
     include_package_data=True,
     scripts=["bin/blabla"],
     zip_safe=False,
+    classifiers=[
+    'Development Status :: 4 - Beta',      # Chose either "3 - Alpha", "4 - Beta" or "5 - Production/Stable" as the current state of your package
+    'Intended Audience :: Healthcare Industry',      # Define that your audience are developers
+    'Topic :: Scientific/Engineering :: Bio-Informatics',
+    'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',   # Again, pick a license
```
Comment (jackanth) [inline setup.py]: Can we get rid of these comments?

File: README.md [hunk 5]
```diff
@@ -19,12 +19,19 @@ For information about contributing, citing, licensing (including commercial lice
...
+
+To install BlaBla from source:
 ```bash
 git clone https://github.com/novoic/blabla.git
 cd blabla
 pip3 install .
 ```
+To install BlaBla using PyPI:
+```bash
+pip install blabla
```
Comment (jackanth) [inline README.md]: `pip` or `pip3`? Either way, these should be the same.

**Pull request merged**